nengo/builder/learning_rules.py

```diff
@@ -461,6 +461,94 @@ def step_simvoja():
         return step_simvoja
 
 
+class SimRLS(Operator):
+    r"""Calculate connection weight change according to the RLS rule.
+
+    Implements the Recursive Least Squares (RLS) learning rule of the form
+
+    .. math::
+
+        g_i &= \sum_j P_{ij}(n-1) r_i
+        P_{ij}(n) &= P_{ij}(n-1)
+                   - g_i g_j / \left( 1 + \sum_{ij} r_i P_{ij}(n-1) r_j \right)
+        \Delta \omega_{ji} &= \frac{\kappa \delta_t}{n} e_j \sum_{k} P_{ik}(n) r_k
```

Comment:
I built the documentation locally and this is what this math looked like:

![Screenshot from 2020-08-04 23-19-08](https://user-images.githubusercontent.com/5420057/89368333-44c80e00-d6a9-11ea-8268-7185d7035941.png)

Not sure why. Maybe something to do with the newline break on 472-473?

Reply:
Fixed. It was missing the `\\` at the end of each line. Not sure why that caused it to not render at all (I would have expected it to all render on one line).